src/main/resources/application.properties

```diff
@@ -25,8 +25,6 @@ ocg.delay=60s
 pras.period=800m
 viper.period=360m
 
-data.deletion.period=24h
-
```

Comment:
Fixed at 24 hrs, no need to have this configurable? - see CsvHousekeepingService